auditbeat/module/auditd/_meta/fields.yml

```diff
@@ -176,35 +179,59 @@
         - name: secondary
           type: keyword
           description: ""
-        - name: selinux
-          type: group
-          description: The SELinux identity of the object.
-          fields:
-          - name: user
-            type: keyword
-            description: The owner of the object.
-          - name: role
-            type: keyword
-            description: The object's SELinux role.
-          - name: domain
-            type: keyword
-            description: The object's SELinux domain or type.
-          - name: level
-            type: keyword
-            example: s0
-            description: The object's SELinux level.
       - name: how
         type: keyword
         description: >
           This describes how the action was performed. Usually this is the exe
           or command that was being executed that triggered the event.
-      - name: key
+
+    - name: paths
```

Comment:
Should this be singular?

Reply:
It is usually a list of raw path objects so I named with a plural. Is that the correct way to do it?

Reply:
My argument is normally to make it singular as long as it's not an array, as the user that is accessing it only looks for a single item and the "key" for this entry reads correct. Here for example `paths.inode`, inode is only one path. But I have this conversation often with @exekias and we probably agreed to disagree 😆